src/main/java/cs361/battleships/models/Game.java

```diff
@@ -42,9 +42,9 @@ public boolean attack(int x, char  y, boolean spaceLaser) {
         if(playerAttack.getResult() != INVALID) {
             playerAttack.setResultClass(opponentsBoard.attack(x, y, spaceLaser));
         }
-        if (playerAttack.getResult() == INVALID) {
-            return false;
-        }
+//        if (playerAttack.getResult() == INVALID) {
```

Comment:
Why was this commented out?

Reply:
It was impossible to make an invalid attack. There are no bounds in the attack function, so sending an attack such as 0, 'N' wouldn't be registered as invalid. (Basically useless code)

Reply:
Okay cool we can probably get rid of it but its fine